packages/ubuntu_welcome/snap/local/launcher

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
do we need these hacks with a non-classic snap? would the gnome extension make sure gtk/glib and friends are correctly setup?